tests/conftest.py

```diff
@@ -20,12 +22,196 @@ def config_values_fixture():
     return config_values
 
 
-@pytest.fixture(autouse=True)
-def test_env(config_values):
-    os.environ = config_values
-    yield
+@pytest.fixture()
+def staff_patron_all_values_dict():
+    return {
+        "FULL_NAME": "Doe, Jane",
+        "OFFICE_ADDRESS": "AA-B1-11",
+        "OFFICE_PHONE": "5555555555",
+        "MIT_ID": "222222222",
+        "EMAIL_ADDRESS": "STAFF_KRB_NAME@MIT.EDU",
+        "APPOINTMENT_END_DATE": datetime.datetime(2023, 6, 30, 0, 0),
+        "KRB_NAME_UPPERCASE": "STAFF_KRB_NAME",
+        "LIBRARY_PERSON_TYPE_CODE": "27",
+        "LIBRARY_PERSON_TYPE": "Staff - Lincoln Labs",
+        "ORG_UNIT_ID": "10000948",
+        "ORG_UNIT_TITLE": "LL-Homeland Protection & Air Traffic Con",
+        "POSITION_TITLE": "Part-time Flexible/LL",
+        "DIRECTORY_TITLE": "Part-time Flexible/LL",
+        "LIBRARY_ID": "33333333333333",
+    }
+
+
+@pytest.fixture()
+def staff_patron_null_values_dict():
+    return {
+        "FULL_NAME": None,
+        "OFFICE_ADDRESS": None,
+        "OFFICE_PHONE": None,
+        "MIT_ID": None,
+        "EMAIL_ADDRESS": "STAFF_KRB_NAME@MIT.EDU",
```

Comment:
The original code made it seem as if the `EMAIL_ADDRESS` field is required in both tables but I want to confirm. I'm emailing the Data Warehouse contact to get a better understanding of what data is required on these 2 tables

Reply:
It's actually `MIT_ID` that's required, I will update the code and push that with any other changes